superset-frontend/src/dashboard/components/PropertiesModal.jsx

```diff
@@ -24,6 +24,7 @@ import Select from 'react-select';
 import AceEditor from 'react-ace';
 import { t } from '@superset-ui/translation';
 import { SupersetClient } from '@superset-ui/connection';
+import stringify from 'json-stringify-pretty-compact';
```

Comment:
what are the advantages to using this module over the native `JSON.stringify` function with formatting (3rd arg)?

Reply:
This generates more compact strings. 

E.g.

```json
{
  "short array": [1, 2, 3],
  "long array": [
    {"x": 1, "y": 2},
    {"x": 2, "y": 1},
    {"x": 1, "y": 1},
    {"x": 2, "y": 2}
  ]
}
```

vs

```json
{
  "bool": true,
  "short array": [
    1,
    2,
    3
  ],
  "long array": [
    {
      "x": 1,
      "y": 2
    },
    {
      "x": 2,
      "y": 1
    },
    {
      "x": 1,
      "y": 1
    },
    {
      "x": 2,
      "y": 2
    }
  ]
}
```

Reply:
ah, hence "compact" in the name 👍